h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -166,6 +169,20 @@ public void revertCoeffNames(String[] colNames, int nclass, TwoDimTableV3 coeffs
           coeffs_table.columns[tableIndex+nclass].name = new String(colNames[tableIndex-1+nclass]);
       }
     }
+    
+    public TwoDimTable build2DTable(double[] ubeta, String[] randomColNames) {
```

Comment:
To strangers, this methods looks like it can build any 2D table. In fact, it contains the random coefficients. For searching in the code, it'd be appreciated to have this method named something like `buildRandomCoefficientsTable`.